nlpia_bot/clibot.py

```diff
@@ -169,8 +169,13 @@ def log_reply(self, statement, reply):
             json.dump(history, f)
 
     def reply(self, statement=''):
+        '''
+        MVP of class methods. Collects and selects from all possible replies from all loaded bots. Logs process. 
+        Returns chosen reply as string.  
+        '''
```

Comment:
Could be a little more concise `""" Collect replies from from loaded bots and return best reply (str). """`

Reply:
This is good. First and last sentence can be deleted, if you like. Also, I like to use `"` for natural language strings intended for humans and `'` for machine-readable strings.